src/components/MainLayout/MainHeader.tsx

```diff
@@ -48,53 +51,60 @@ export const MainHeader = () => {
     <Box
       css={{
         px: '$1xl',
-        pt: '9px', // manual offset to align breadcrumb
-        height: '80px',
+        height: '91px',
```

Comment:
magic number not ideal, should add it to theme.sizes under `headerHeight` or something to that effect

Reply:
I agree with @META-DREAMER that it's good to explain magic numbers, but I don't think this should be put into the theme, because it's unlikely to be reused outside of this component. It could be a const declared at the top of this file with an explanatory comment